tools/styles/tokens/express/system-theme-bridge.css

```diff
@@ -630,13 +630,17 @@
     --system-tag-size-large-clear-button-spacing-inline-end: var(--spectrum-component-pill-edge-to-visual-200);
     --system-textfield-background-color: var(--spectrum-gray-50);
     --system-textfield-background-color-disabled: var(--spectrum-disabled-background-color);
-    --system-textfield-border-color: var(--spectrum-gray-400);
-    --system-textfield-border-color-hover: var(--spectrum-gray-500);
+    --system-textfield-border-color: var(--spectrum-gray-500);
+    --system-textfield-border-color-hover: var(--spectrum-gray-600);
     --system-textfield-border-color-focus: var(--spectrum-gray-800);
     --system-textfield-border-color-focus-hover: var(--spectrum-gray-900);
     --system-textfield-border-color-keyboard-focus: var(--spectrum-gray-900);
     --system-textfield-border-color-disabled: var(--spectrum-gray-200);
     --system-textfield-border-width: var(--spectrum-border-width-200);
+    --system-textfield-spacing-block-start: var(--spectrum-component-top-to-text-100);
+    --system-textfield-spacing-block-start-small: var(--spectrum-component-top-to-text-50);
```

Comment:
Is the small spacing here supposed to be top-to-text-50 or 75? 

Most of the component specs follow 75/100/200/300, but since I don't have access to the Express Figma files, this could totally be right! I only noticed it because it didn't follow that pattern.